test/docker/java8/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM openjdk:8-jdk-slim AS build
+
+RUN apt-get update \
+    && apt-get install -y git
+
+# Build the source from this repository
+RUN mkdir -p /build
+
+WORKDIR /appmap
+
+ADD gradle              ./gradle
+ADD gradlew             ./
+ADD settings.gradle     ./
+ADD build.gradle        ./
+ADD src                 ./src
+ADD buildSrc            ./buildSrc
+
+RUN ./gradlew --no-daemon build  \
+    && mv build/libs/appmap-*-java*.jar /build/appmap.jar
+
+FROM openjdk:8-alpine
+LABEL maintainer="Dustin Byrne <dustin@app.land>"
+
+RUN apk add --no-cache -U bash curl git jq ncurses less
+
+RUN git clone https://github.com/bats-core/bats-core.git \
+    && git clone https://github.com/bats-core/bats-support.git /test/test_helper/bats-support \
+    && git clone https://github.com/bats-core/bats-assert.git /test/test_helper/bats-assert \
+    && cd bats-core \
+    && ./install.sh /usr/local
+
+COPY --from=petclinic:latest /build /
```

Comment:
Isn't the petclinic jar file in the `petclinic:latest` image going to be built with Java 11? Is it going to work here?

Reply:
It will work as spring-petclinic has  java version specified in [pom.xml](https://github.com/spring-projects/spring-petclinic/blob/main/pom.xml) . To demonstrate,  [this](https://travis-ci.com/github/applandinc/appmap-java/jobs/397452174) job executed java 8 docker configuration successfully.